Library/Formula/haskell-stack.rb

```diff
@@ -4,9 +4,9 @@ class HaskellStack < Formula
   include Language::Haskell::Cabal
 
   desc "The Haskell Tool Stack"
-  homepage "https://www.stackage.org"
-  url "https://hackage.haskell.org/package/stack-0.1.6.0/stack-0.1.6.0.tar.gz"
-  sha256 "a47ffc204b9caef8281d1e5daebc21bc9d4d2414ed695dc10d32fcca4d81978d"
+  homepage "https://github.com/commercialhaskell/stack"
```

Comment:
That was a historical detail (an earlier incarnation of stack was called
stackage-build before the two projects were separated), and as of now no
longer does.  The `stack.cabal` file has pointed to
https://github.com/commercialhaskell/stack for a long time, as do the
`.deb`s and `.rpms`.

>